flask/app/models.py

```diff
@@ -421,25 +424,31 @@ class Variation(str, Enum):
     Stop_Lost = "Stop_Lost"
     Start_lost = "Start_lost"
     Protein_Altering_Variant = "Protein_Altering_Variant"
-    Start_Retrained_Variant = "Start_Retrained_Variant"
+    Start_Retained_Variant = "Start_Retained_Variant"
     Mature_miRNA_Variant = "Mature_miRNA_Variant"
-    Stop_Retained_Variant = "Stop_Retained_Variant"
 
 
+@dataclass
 class Gene(db.Model):
     gene_id: int = db.Column(db.Integer, primary_key=True)
-    hgnc_gene_id: int = db.Column(db.Integer)
+    hgnc_gene_id: int = db.Column(db.Integer, unique=True)
     ensembl_id: int = db.Column(db.Integer)
+    gene: str = db.Column(db.String(50))
```

Comment:
What does this represent now?

Reply:
The `gene` column in the reports which are not necessarily hgnc gene names (unfortunately)

Reply:
after giving it some thought, here are a few options we can take to have a more meaningful ‘gene’ column (since it is the goal of this feature after all)
- query the variant position against an external resource to obtain the hgnc gene name/ensembl id/etc. unfortunately i think is easier said than done unless someone knows of a resource where all this information is single handedly kept? it's also a bit roundabout since i believe this is what was done in the first place to generate the report
- take the existing gene column and attempt to map it back to the same identifiers above (hgnc gene name/id/ensembl id, etc). also easier said than done 
- leave the gene column and assume it is a single source of truth since it was generated from the reports

thoughts?

Reply:
Should the ensembl_id also be unique?